hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -123,14 +129,19 @@ message Pipeline {
     optional ReplicationType type = 3 [default = STAND_ALONE];
     optional ReplicationFactor factor = 4 [default = ONE];
     required PipelineID id = 5;
+    // TODO: Deprecate this and replace with leaderDnID
     optional string leaderID = 6;
     repeated uint32 memberOrders = 7;
     optional uint64 creationTimeStamp = 8;
+    // TODO: Deprecate this and replace with suggesterLeaderDnID
     optional UUID suggestedLeaderID = 9;
     repeated uint32 memberReplicaIndexes = 10;
     optional ECReplicationConfig ecReplicationConfig = 11;
-    // TODO(runzhiwang): when leaderID is gone, specify 6 as the index of leaderID128
+
     optional UUID leaderID128 = 100;
+    // TODO: Replace UUID with DatanodeIDProto
+    optional DatanodeIDProto leaderDnID = 101;
+    optional DatanodeIDProto suggesterLeaderDnID = 102;
```

Comment:
Sorry, I just noticed a typo:

```suggestion
    optional DatanodeIDProto leaderDnID = 101;
    optional DatanodeIDProto suggestedLeaderDnID = 102;
```

I would also consider expanding `...DnID` to `...DatanodeID`.

Reply:
My bad, I will fix it.